frontend/plugins/operation_ui/src/modules/task/components/NotesField.tsx

```
@@ -0,0 +1,22 @@
+  const { activities } = useActivities(taskId);
+  return (
+    <div className="flex flex-col gap-4 mb-14">
+      {activities &&
+        activities
+          ?.filter((activity) => activity.module === 'NOTE')
+          .map((activity) => (
+            <NoteInputReadOnly
+              newValueId={activity.metadata?.newValue}
+              authorId={activity.createdBy}
+              createdAt={activity.createdAt?.toLocaleString()}
+            />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)